src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/AsylumCaseFieldDefinition.java

```diff
@@ -377,7 +377,10 @@
         "interpreterLanguageReadonly", new TypeReference<List<IdValue<InterpreterLanguage>>>() {}),
 
     SUBMIT_HEARING_REQUIREMENTS_AVAILABLE(
-        "submitHearingRequirementsAvailable", new TypeReference<YesOrNo>() {})
+        "submitHearingRequirementsAvailable", new TypeReference<YesOrNo>() {}),
+
+    UPDATE_HEARING_REQUIREMENTS_EXISTS(
+        "updateHearingRequirementsExists", new TypeReference<String>() {})
```

Comment:
It would be good to have prefix "application" for the flag. There is for all Application related flags. It is minor thing but still helpful to manage all those flags.

Reply:
prefixed the flag with 'application'